docs/administration/github-org.rst

```diff
@@ -1,10 +1,102 @@
+###################
 GitHub organization
-===================
+###################
 
 This page explains how the `RITlug GitHub organization`_ is organized and how
-it is intended to be used.
+to manage it.
 
-.. _`RITlug GitHub organization`: https://github.com/RITlug
 
-…more to come soon.
+*******
+Purpose
+*******
+
+The GitHub organization is a central place to manage club projects, an
+approximation of membership, and host some of our infrastructure. Most data is
```

Comment:
Change wording a little.

"The GitHub organization is a central place to manage club projects, membership, and infrastructure."

This allows more concise wording. If you want any other changes feel free to ping me.